network/internal/p2pfixtures/fixtures.go

```diff
@@ -5,6 +5,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
```

Comment:
Nothing changed in this file, just some functions re-ordered due to move and revert-move.